src/main/java/sda/java/loginproject/entity/Stock.java

```diff
@@ -0,0 +1,65 @@
+package sda.java.loginproject.entity;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Table(name = "Stock")
+public class Stock {
+    private static final String STOCK_SEQUENCE = "stock_id_seq";
+    private static final String STOCK_GENERATOR = "stock_generator";
+
+    @Id
+    @SequenceGenerator(name = STOCK_GENERATOR , sequenceName = STOCK_SEQUENCE )
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = STOCK_GENERATOR )
+    private int id;
+
+    @Column(name = "nrPieces")
```

Comment:
multiple words in database should always have _ as the separator, not camel case

Reply:
done, changed nrPieces column name